### ISSUE TITLE
Wemo Maker must be triggered as accessory in Homebridge before it works in Apple Home

### ISSUE THREAD

**Issue by ElderoftheInternet** [rudders/homebridge-wemo#46]
### What issue do you have? Please be as thorough and explicit as possible.
My Wemo Maker appears as not responding in Apple Home. However it works fine as an accessory in Homebridge. In fact, once I use it once as an accessory, in this case to open/close my garage door it then works fine in Apple Home and the not responding message disappears from there.

### Details of your setup.
* Do you use Homebridge (with config-ui?) or HOOBS? 
Homebridge with config-ui

* Which version of Homebridge/HOOBS do you have?
v.1.4.0

* Which version of this plugin (homebridge-wemo) do you have? Has the issue started since upgrading from a previous version?
homebridge-wemo v4.5.4

* Which Wemo devices do you have that are causing issues? Please include product models if applicable.
Wemo Maker 
WeMo_WW_2.00.11423.PVT-OWRT-Maker
Hardware version v1

### Please paste any relevant logs below.

```

```
There is nothing in the logs other than it starts successfully.

[25/01/2022, 13:35:23] Homebridge v1.4.0 (HAP v0.10.0) (Homebridge 1688) is running on port 51305.
[25/01/2022, 13:35:24] [Wemo] [Garage] initialising with options {"logging":"standard"}.
[25/01/2022, 13:35:24] [Wemo] [Garage] initialised with s/n 221444S0000200 and ip/port 192.168.1.5:49153
[25/01/2022, 13:35:24] [Wemo] [Garage] http has been established.
[25/01/2022, 13:35:25] [Wemo] ✓ Setup complete. Join the plugin's Discord community! https://discord.gg/cMGhNtZ3tW
[25/01/2022, 13:37:31] [Wemo] [Garage] current state [on].
[25/01/2022, 13:38:04] [Wemo] [Garage] current state [off].
[25/01/2022, 13:38:17] [Wemo] [Garage] current state [on].

Config (I tried to setup as Garage door and didn't resolve issue)
        {
            "name": "Wemo",
            "mode": "auto",
            "debug": false,
            "discoveryInterval": 60,
            "upnpInterval": 600,
            "disableUPNP": true,
            "makerTypes": [
                {
                    "label": "Garage",
                    "makerType": "garageDoor",
                    "makerTimer": 10
                }
            ],
            "platform": "Wemo"
        },

**Comment by bwp91** [comment]
Hi @ElderoftheInternet 

Your config needs to include the serial number of the wemo maker:

```
"makerTypes": [
  {
    "serialNumber": "221444S0000200",
    "label": "Garage",
    "makerType": "garageDoor",
    "makerTimer": 10
  }
],
```

**Comment by bwp91** [comment]
Are you able to update to the beta version of the plugin? I've made a couple of small changes which might fix this continued 'no response' in the Home app.

The no response should be initially shown when you restart the plugin until the device has been found and a status update is requested.

It might take a couple of restarts for this to properly take effect.

**Comment by ElderoftheInternet** [comment]
Thanks Ben. Appreciate the follow-up. Have added the serial number to the
config file and updated to the latest beta.

I've restarted homebridge a number of times but I seem to be getting a new
message in the homebride log as per below:

[27/01/2022, 09:54:27] [Wemo] [Garage] initialising with options
{"logging":"standard","makerTimer":5}.
[27/01/2022, 09:54:27] [Wemo] [Garage] could not be initialised as Cannot
read properties of undefined (reading 'cacheLastTargetState') at new
deviceMakerGarage
(/usr/local/lib/node_modules/homebridge-wemo/lib/device/maker-garage.js:103:30).

[27/01/2022, 09:54:29] [Wemo] ✓ Setup complete. Join the plugin's Discord
community! https://discord.gg/cMGhNtZ3tW

[27/01/2022, 09:54:27] [Wemo] [Garage] initialising with options
{"logging":"standard","makerTimer":5}.
[27/01/2022, 09:54:27] [Wemo] [Garage] could not be initialised as Cannot
read properties of undefined (reading 'cacheLastTargetState') at new
deviceMakerGarage
(/usr/local/lib/node_modules/homebridge-wemo/lib/device/maker-garage.js:103:30).
[27/01/2022, 09:55:27] [Wemo] [Garage] initialising with options
{"logging":"standard","makerTimer":5}.
[27/01/2022, 09:55:27] [Wemo] [Garage] could not be initialised as Cannot
read properties of undefined (reading 'cacheLastTargetState') at new
deviceMakerGarage
(/usr/local/lib/node_modules/homebridge-wemo/lib/device/maker-garage.js:103:30).
[27/01/2022, 09:56:27] [Wemo] [Garage] initialising with options
{"logging":"standard","makerTimer":5}.
[27/01/2022, 09:56:27] [Wemo] [Garage] could not be initialised as Cannot
read properties of undefined (reading 'cacheLastTargetState') at new
deviceMakerGarage
(/usr/local/lib/node_modules/homebridge-wemo/lib/device/maker-garage.js:103:30).
[27/01/2022, 09:57:27] [Wemo] [Garage] initialising with options
{"logging":"standard","makerTimer":5}.
[27/01/2022, 09:57:27] [Wemo] [Garage] could not be initialised as Cannot
read properties of undefined (reading 'cacheLastTargetState') at new
deviceMakerGarage
(/usr/local/lib/node_modules/homebridge-wemo/lib/device/maker-garage.js:103:30).
[27/01/2022, 09:58:27] [Wemo] [Garage] initialising with options
{"logging":"standard","makerTimer":5}.
[27/01/2022, 09:58:27] [Wemo] [Garage] could not be initialised as Cannot
read properties of undefined (reading 'cacheLastTargetState') at new
deviceMakerGarage
(/usr/local/lib/node_modules/homebridge-wemo/lib/device/maker-garage.js:103:30).
[27/01/2022, 09:59:27] [Wemo] [Garage] initialising with options
{"logging":"standard","makerTimer":5}.
[27/01/2022, 09:59:27] [Wemo] [Garage] could not be initialised as Cannot
read properties of undefined (reading 'cacheLastTargetState') at new
deviceMakerGarage
(/usr/local/lib/node_modules/homebridge-wemo/lib/device/maker-garage.js:103:30).

Thanks

Dean

**Comment by bwp91** [comment]
@ElderoftheInternet oops can you update to new beta and try again?

**Comment by ElderoftheInternet** [comment]
Thanks Ben. All working now. Appreciate your help with this!

Dean

**Comment by bwp91** [comment]
Just released v4.6.0 with the fixes includes! Will close this issue now.
